src/components/embed/ui/organisms/account-selector/AccountSelector.tsx

```diff
@@ -59,7 +57,7 @@ export function AccountSelector({
         content={
           <>
             {wallets.map((wallet, id) => (
-              <DropdownItem key={id} onClick={handleAccountClick}>
```

Comment:
Bug, but we didn't notice because `setActiveWallet()` had another bug, so it would just throw an error instead of incorrectly unsetting the active wallet.